warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java

```diff
@@ -260,11 +260,21 @@ public int run(String[] args) throws Exception {
         log.setLevel(Level.INFO);
         
         // Initialize the markings file helper so we get the right markings file
+        long start = System.currentTimeMillis();
```

Comment:
I know we've used these timings-in-code in other places before, so there's a precedent but do you think we might have more success just profiling a few jobs instead of adding timing code? I feel like it adds clutter to the code and makes it harder to follow.

Reply:
I will have to concur with @jschmidt10 here in that this is really cluttering up the code.  Using the StopWatch class may make this clean enough to accept.